app/validators/income_assessment/answers_validator.rb

```diff
@@ -82,6 +82,7 @@ def manage_without_income_complete?
       return true if record.income_payments.present? || record.income_benefits.present?
```

Comment:
Can we update this method to use the same logic as used in the decision tree (see note in review) above.